app/models/index.js

```diff
@@ -4,7 +4,7 @@ const fs = require('fs'),
   basename = path.basename(__filename),
   config = require('../../config'),
   dbConfig = require('../../config/db')[config.environment],
-  db = {};
+  db = { models: {} };
```

Comment:
I dont think there is any need to create the property here. If I do:
```
const a  = {};
a.model = myModel;
```
then `a` will have the property created